### ISSUE TITLE
Flaky test: docker_integration_test.go

### ISSUE THREAD

**Issue by ycombinator** [elastic/beats#11627]
## Flaky Test

* **Test Name:** docker_integration_test.go
* **Link:** https://github.com/elastic/beats/blob/master/libbeat/autodiscover/providers/docker/docker_integration_test.go#L116
* **Branch:** `master` (e.g. https://beats-ci.elastic.co/job/elastic+beats+master+multijob-linux/918/beat=libbeat,label=linux-immutable/console)

### Stack Trace

```
03:22:42 command [go test -race -tags integration -cover -coverprofile /tmp/gotestcover-021614182 github.com/elastic/beats/libbeat/autodiscover/providers/docker]: exit status 1
03:22:42 --- FAIL: TestDockerStart (13.56s)
03:22:42     docker_integration_test.go:116: Timeout waiting for provider events
03:22:42 FAIL
```

**Comment by exekias** [comment]
I tested this locally and seems to work, travis seems to pass them, will need to investigate jenkins

**Comment by ruflin** [comment]
Potentially related to docker version used? Perhaps an update / downgrade happened on the workers (infra will know). I'm surprised that it happened from one day to the other.

**Comment by exekias** [comment]
closed by #11721
